pyramid_swagger/tween.py

```diff
@@ -248,6 +244,12 @@ def files(self):
     def json(self, **kwargs):
         return getattr(self.request, 'json_body', {})
 
+    def __getattr__(self, name):
```

Comment:
So if I'm reading the [Object Model](https://docs.python.org/2/reference/datamodel.html#object.__getattr__) properly, this will only get used when we are using a name which hasn't already been defined in this class? i.e. the json attribute will hit the method two lines above, not pass through this getattr func?